process/subprocesses/ghsci.py

```diff
@@ -1136,45 +1130,59 @@ def _create_open_space_areas(self):
 
         open_space_areas_setup(self.codename)
         return 'Open space areas created.'
+    
+    def _compute_lpugs(self):

```

Comment:
This is good!  Although, add checks in these functions to see if GEE is configured as true, and that the process has been launched using the correct launcher and if not alert the user that this must be done.   Perhaps an efficient way to do this, since it will be checked at multiple points, could be to add a subfunction to ghsci object that 1) checks configuration of gee, and 2) tests import of some small but crucial library only present in the GEE-specific docker image (or some other simpler way you can think of that will test that GEE libraries are accessible for use).   That `ghsci.check_gee()` function could then be used any time a check is required if GEE analysis is to proceed.

Reply:
just for clarity -- while I think the concept of having a function to launch ee indicator workflow may be okay here, I would recommend it isn't simply 'lpugs' but rather the ee workflow itself.  Later the indicators included within that workflow would be configured within indicators.yml (that is more consistent with current approach, where we don't e.g. have _computer_walkability_index or other single indicator specific functions)